scripts/curvefit_test.py

```diff
@@ -0,0 +1,227 @@
+import sys
```

Comment:
curvefit_test.py and make_shift_calib.py have essentially the same issues:
* Make them executable (ie, put `#!/usr/bin/env python2` at the top
* Put a short comment explaining what the script is for and what it does. Finish with an example command line to show how to call the script
* This scripts are very specific to the nikonc. So either modify the script names to highlight they are not (for example) general curve-fitting, but very specific one for confocal mirror movements. Or you could just move the scripts to the nikonc driver repo (and still rename to `curvefit_mirror.py` or something of this style).